sorting/bubble_sort_recursion.c

```diff
@@ -0,0 +1,58 @@
+#include <assert.h>
```

Comment:
Please update the comment block here using the [guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program)